scraper/core/server.py

```diff
@@ -93,9 +92,9 @@ def collect_data_from_sources():
                 Scraper.refetch_sources()
 
             data_list = [
-                    #*Scraper.__get_data_from_rss_feed(),] 
-                    *Scraper.__get_data_from_twitter_feed(),] 
-                    #*Scraper.__get_data_from_api()]
+                    *Scraper.__get_data_from_rss_feed(),
+                    *Scraper.__get_data_from_twitter_feed(),
+                    *Scraper.__get_data_from_api()]
```

Comment:
Is the API service working?

Reply:
Still getting errors, but it is returning values. I commented it out for the moment and am going to create a new issue.

Reply:
Fine.